tests/unit/lint/lint-test.js

```diff
@@ -0,0 +1,5 @@
+import { module, test } from 'qunit'; // eslint-disable-line
+
+/**
+ * Linting CI tests will be injected here
+ */
```

Comment:
Yeah I tried to keep as much of the babel settings in the `.babelrc` The only thing I tried to keep in the `package.json` is input and outputs because it makes it easier to walk through what babel is building.